utils/create_cluster/local_cluster.py

```diff
@@ -65,7 +65,7 @@ def create_cluster(n_nodes, port):
     # Start servers
     host = '127.0.0.1'
     redis = os.getenv('REDIS_INSTALL_PATH') + '/redis-server'
-    redisai = os.getenv('REDISAI_CPU_INSTALL_PATH') + '/redisai.so '
+    redisai = os.getenv('REDISAI_GPU_INSTALL_PATH') + '/redisai.so '
```

Comment:
Do we need some kind of switch here depending on the test device to choose between ``REDISAI_CPU_INSTALL_PATH`` and ``REDISAI_GPU_INSTALL_PATH``?

Reply:
Thanks for catching that. I've added a line to check for `SMARTREDIS_TEST_DEVICE`, defaulting to 'cpu' if it's not set.